server/src/config.js

```diff
@@ -9,6 +9,7 @@ dotenv.config({path: path.resolve(__dirname, `../.env.${process.env.NODE_ENV}`)}
 
 const config = {
   jwtSecret: process.env.JWT_SECRET,
+  enableVerificationPublishing: process.env.ENABLE_VERIFY_PUBLISHING
 };
 
-module.exports = config;
+export { config };
```

Comment:
Just a reminder, sometimes we got errors using export syntax.

Reply:
I had hard time fixing the error when using `module.exports`. The problem is we use Es6 syntax in this project and mixing it with commonjs causes weird errors.